openpype/settings/defaults/project_settings/ftrack.json

```diff
@@ -494,4 +495,4 @@
             "farm_status_profiles": []
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Please use Settings UI to save defaults.

Reply:
How do I do that?
Just saving commit to the database rather than the defaults `.json` file.

Reply:
https://openpype.io/docs/dev_settings#how-to-add-new-settings

Reply:
Thanks!